gsuite/client.py

```diff
@@ -0,0 +1,49 @@
+import os
+import httplib2
+
+from apiclient import discovery
+from oauth2client.file import Storage
+
+ADMIN_DIRECTORY_USER_READONLY = 'admin-directory-user-readonly'
+CREDENTIALS = [
+    (ADMIN_DIRECTORY_USER_READONLY, 'https://www.googleapis.com/auth/admin.directory.user.readonly'),
+]
+
+
+def get_credential_dir():
+    home_dir = os.path.expanduser('~')
+    credential_dir = os.path.join(home_dir, '.credentials')
```

Comment:
this might conflict with the github org scripts filename (should be fine for now)